Frends.AzureBlobStorage.DeleteBlob/Frends.AzureBlobStorage.DeleteBlob/DeleteBlob.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+
+namespace Frends.AzureBlobStorage.DeleteBlob
+{
+    /// <summary>
+    /// Task class.
+    /// </summary>
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        /// Deletes a single blob from Azure blob storage.
+        /// [Documentation](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/tree/main/Frends.AzureBlobStorage.DeleteBlob)
+        /// </summary>
+        /// <param name="target">Blob which will be deleted.</param>
+        /// <param name="connectionProperties">Connection options.</param>
+        /// <returns>Object { bool Success }</returns>
+        public static async Task<Result> DeleteBlob(
+            [PropertyTab] Input target,
+            [PropertyTab] Options connectionProperties,

```

Comment:
I think parameter names could be a bit better

Reply:
Suggestions are appreciated. Naming is not my strong side.

Reply:
I am even worse, but Input input and Options options would be fine. Or at least Options options, since you also moved the conn string into input.

Reply:
Made the changes.